proto/store/setting.proto

```diff
@@ -79,3 +79,21 @@ message SMTPMailDeliverySetting {
   // The sender email address.
   string from = 10;
 }
+
+message AppIMSetting {
+  enum IMType {
+    IM_UNSPECIFIED = 0;
+    IM_FEISHU = 1;
+  }
```

Comment:
enums should be PascalCase, which is ImType in this case.

The values should be `IM_TYPE_FEISHU`

Reply:
yeah.. I found the docs here https://protobuf.dev/programming-guides/style/#enums
sadly seems many of our enums don't follow the style guide

Reply:
We can use aip.dev

Reply:
I see https://google.aip.dev/126

Reply:
Anyway, if it works, let's do it.